wemake_python_styleguide/visitors/ast/exceptions.py

```diff
@@ -1,12 +1,13 @@
 import ast
 from collections import Counter
-from typing import ClassVar, Set, Tuple
+from typing import ClassVar, Set
 
 from typing_extensions import final
 
 from wemake_python_styleguide.compat.aliases import FunctionNodes
 from wemake_python_styleguide.logic import nodes
 from wemake_python_styleguide.logic.tree import exceptions
+from wemake_python_styleguide.logic.tree.exceptions import find_returning_nodes
```

Comment:
We have `exceptions` and `from exception` imports. Why so?

There's a proposed rule to forbid this style.